tech-committee-charter.md

```diff
@@ -37,7 +37,6 @@ The TC is responsible for all technical development within the OpenTelemetry pro
 * Setting release dates
 * Quality standards for releases
 * Technical direction
-* GitHub repository management, membership and hosting
```

Comment:
No objection, but I'm wondering if you can share some of the reasoning publicly.

Reply:
We wanted to both extend (by including GC members) and reduce (by limiting to now only a subset of GC+TC, see #2749) the number of people having admin access to the Github org.